docs/1-user-guide.md

```diff
@@ -93,7 +93,7 @@ For updates on your requests, you can either poll using the `get` (`/tranasction
 5. Under `Docker Image Tag`, select either `latest` or `nightly`
    ![Alt text](./images/Zeet-Docker-Source-Setting.png)
 6. Select the Target cluster you want the above image to be deployed to
-7. Choose your `Compute` settings
+7. Choose your `Computer` settings
```

Comment:
It's actually `Compute` Setting

Reply:
<img width="1039" alt="image" src="https://github.com/thirdweb-dev/engine/assets/132962163/5443d1ef-696b-4ea0-afde-e81e3249c9c6">